README.md

```diff
@@ -1,10 +1,122 @@
+See the [recipes](recipes/)
+
+## Contributing
+
+### General Requirements
+
+For information about contributing to this repo, code of conduct guidelines, etc., see the community [CONTRIBUTING][CG] and [Code of Conduct][CoC] guides.  All commits require DCO-signoff (discussed [here][CG-legal]) _and_ GPG or SSH signing (discussed [here][CG-signing]).  The GitHub recommended code security settings are enforced on this public repository (which include the signing requirement).
+
+### What is a "Recipe"?
+
+A "cookbook" is composed of "recipes".
+In this version of the Granite Timeseries Cookbook, a "recipe" is a Python notebook.
+
+Under the implied cooking analogy, there are three key defining elements of a
+good recipe:
+
+1. It clearly states the required ingredients and tools up front
+2. It is straightforward to reproduce efficiently
+3. It results in something delicious to eat
+
+The "ingredients" and "tools" of the first point mean the data and software at hand.
+
+The second point is straightforward to map into the technical space:
+The reader should be able to run the cells of the notebook sequentially
+and the same result as what was published in the original notebook.
+Our objective is to make these recipes reproducible in 15 minutes or less.
+This may inform the decisions about recipe granularity.
+
+The third point is more subtle, and is what sets this kind of writing apart.
+In a technical sense, "something delicious to eat" means a system
+that demonstrates useful functionality in such a way that it sets the
+reader on the path to adopting it in their environment.  It clearly articulate the business value achieved by the resulting system.
+
+For an "Energy Demand Forecasting" recipe, for instance, the recipe should:
+
+- State at a high level how a forecasting capability could create "business value", and therefore be worth the investment.
+- Provide example input data
+- Provide an example forecast request
+- Show the expected resulting prediction
+- Provide enough code to walk through the whole process
+
+For recipe authors with strong familiarity with a specific capability or tool,
+the first inclination may be to write a recipe oriented around the tool.
+Consider alternate ways to phrase the recipe so that the end result is showcased, rather than the tool.
+
+Under the cooking analogy, that would mean writing a great soup recipe rather than one that talks about the features of a food processor.  If the soup tastes great and is easy to prepare, the reader will likely want to know more about how it was made.
+
+Recipes will vary in complexity.
+Some may be single inference calls.
+Others may illustrate useful agentic workflows.
+
+A "cookbook" is not intended to be a comprehensive guide to all
+issues that may arise during development with Granite Timeseries.
+Recipes will link to helpful external resources on topics including: distributed systems, UI, design, AI/ML theory, metrics, etc.
+
+### Exceptions
+
```

Comment:
Should some of the TS content be in a separate README for the TS recipes?

Reply:
Possibly; what TS content do you have in mind?

Reply:
@deanwampler I'm merging this PR and will address your comment in a later PR, once I clarify what you are looking for.